Add receptorctl reload doc (#1049)
ansible/receptor: docs/source/receptorctl/receptorctl_reload.rst

------
reload
------

.. contents::
   :local:

> ``receptorctl reload`` Reload configuration on the Receptor node to which it is connected.

Reviewer: ```suggestion
``receptorctl reload`` reloads the Receptor configuration for the connected node.
```

Reviewer: Just noticed we're being inconsistent with the capitalization in the description. Most of the docs are lower-case but version is an exception "Displays": https://github.com/ansible/receptor/blob/devel/docs/source/receptorctl/receptorctl_version.rst
